src/main/java/com/faforever/client/status/Incident.java

```diff
@@ -0,0 +1,22 @@
+package com.faforever.client.status;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+import java.time.OffsetDateTime;
+
+@Value
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Incident {
```

Comment:
Do you have a suggestion for a class name that represents [the `Incident` API object](https://docs.statping.com/#folder-incidents), other than `Incident`? We could call it `StatusPingIncident` but that's rather redundant and an anti-pattern because the package makes it clear what this class is here for.

I could add a Javadoc but other than "Represents an Incident" and a link to https://docs.statping.com/#folder-incidents I wouldn't know what to explain.

Reply:
Alright, I gave it a try